outli.el

```diff
@@ -97,7 +99,8 @@ matching mode are used."
 				      (const :tag "No Styling" 'none)
 				      (const :tag "Uniform Style" t)
 				      (const :tag "Default Style" nil))
-			      (boolean :tag "Omit Overline")))))
+			      (boolean :tag "Omit Overline")
+                              (natnum :tag "No. Levels")))))
```

Comment:
I'm not convinced we need to be able to configure this separately either, since `outline-heading-levels` is a local variable, you could just set that in a mode hook, for example.